src/core/gui/PageView.cpp

```diff
@@ -644,10 +644,6 @@ auto XojPageView::onButtonReleaseEvent(const PositionInputData& pos) -> bool {
         if (this->getPage()->getPdfPageNr() != npos) {
             auto page = pdfDoc.getPage(this->getPage()->getPdfPageNr());
 
-            Layer* layer = this->page->getSelectedLayer();
-            const Layer::Index layerId = this->page->getSelectedLayerId();
-            const bool isBackgroundLayer = (layerId == 0);
```

Comment:
Dead code ?? Are there any side effects of those lines? I think it safe to remove.

Reply:
They can be removed indeed. It looks like I forgot to remove those lines in #4673.